core-bundle/src/Resources/contao/templates/search/search_default.html5

```diff
@@ -1,8 +1,13 @@
 
 <div class="<?= $this->class ?>">
 
-  <h3><a href="<?= $this->href ?>" title="<?= $this->title ?>"><?= $this->link ?></a> <span class="relevance">[<?= $this->relevance ?>]</span></h3>
-
+  <a href="<?= $this->href ?>" title="<?= $this->title ?>">
```

Comment:
Is this a good default? Guess most people will likely want to position the image next to the search results but I don't know which variant we should choose to provide the best styling flexibility?

Reply:
I think I would keep the `<h3>` as it was before and add the image with its own `<a>` before the headline. 

We could even set the URL in the figure builder with `$figureBuilder->setLinkHref($result['url'])` in `ModuleSearch:355`.

Reply:
Updated in 49ef4f677061c7d114c9ef6a591fcf248d8e6827.